backend/minors/REL.yaml

```diff
@@ -0,0 +1,150 @@
+---
+type: Minor
+name: Religion
+code: REL
+degree:
+urls:
+- https://religion.princeton.edu/undergraduate/religion-minor
+description: The purpose of the Religion Minor is to provide an interdisciplinary framework in which students majoring in other departments can synthesize their coursework in religious studies to most effectively enrich their other academic work. This framework orients students’ personal, pre-professional, and scholarly interests in thinking about religion toward comparative cultural and social contexts. Students who complete the minor will leave Princeton with a better understanding of the artistic, economic, ethical, historical, philosophical, and political effects of religious traditions and religious thought.
+excluded_major: 
+- REL
+contacts:
+- type: Director of Undergraduate Studies
+  name: Bryan Lowe
+  email: bdlowe@princeton.edu
+req_list:
+- name: REL Minor Courses
+  max_counted: 1
+  min_needed: 2
+  explanation: |-
+    Students must successfully complete five courses related to religion, fulfilling certain conditions.
+  double_counting_allowed: false
+  req_list:
+  - name: REL Courses
+    max_counted: 1
+    min_needed: 5
+    explanation: |-
+      At least four of these courses must be offered by the Department of Religion. One course may be 
+      offered elsewhere in the University, as long as it is approved as suitably relevant by the DUS. 
+      Courses cross-listed with REL may be considered pre-approved.
+    double_counting_allowed: false
```

Comment:
We prob want double_counting to be true, so courses can satisfy the 5-course req and the "Areas of Study"